server/plugin.go

```diff
@@ -137,15 +161,15 @@ func (p *Plugin) verifyJenkinsCredentials(username, token string) (bool, error)
 
 // createEphemeralPost creates an ephemeral post
 func (p *Plugin) createEphemeralPost(userID, channelID, message string) {
+	botUserID, _ := p.ensureBot()
```

Comment:
Calling this every time we want to use the bot account, not sure about performance implications. I tried storing it in the `*configuration` struct, but it kept getting overwritten in `onConfigurationChange` / `p.setConfiguration`

Reply:
Why not store it in the `Plugin` struct?